### ISSUE TITLE
Can we change predict_len if we use TTM model ?

### ISSUE THREAD

**Issue by JieNi4** [ibm-granite/granite-tsfm#102]
Pretty good model! There are two models available (512-96 and 1024-96). The predict_len is set to 96, can we change it?

**Comment by wgifford** [comment]
Hi @JieNi4 thanks for your interest.

You can set `prediction_filter_length` to a value less than `prediction_length` and the model will only provide that many outputs. The loss will also be calculated based on `prediction_filter_length`; during fine tuning you can tune the existing head based on this shorter output.

**Comment by JieNi4** [comment]
Thanks! Can we add multi-variance into zero-shot or fine-tuning(few-show) process? I have seen it's possible in your paper. Thank you very much for your reply.

**Comment by wgifford** [comment]
Hi @JieNi4 yes -- if you fine-tune you can freeze the backbone, then enable channel mixing in the decoder and tune.

Some code snippets:
```
finetune_model = TinyTimeMixerForPrediction.from_pretrained(
    "ibm-granite/granite-timeseries-ttm-v1",
    num_input_channels=<num channels for your dataset>,
    decoder_mode="mix_channel",
)

# freeze parameters in the model backbone
for param in finetune_model.backbone.parameters():
    param.requires_grad = False

# define trainer
<standard training arguments, trainer definition>

trainer.train()
```

**Comment by hwanyu112** [comment]
Since the `forecast_length` of TTM is set to 96, how to get the full zero shot evaluation results of different forecast lengths?
